how-to/domains-ssl-you.md

```diff
@@ -0,0 +1,7 @@
+# :rocket: Domains, SSL, & YOU!
+
+> This document is private and hosted in another repository. Please follow the link below to access it.
```

Comment:
To me, it gives the feeling of something foreign or external. I don't have a strong opinion one way or the other.